src/components/CioQuiz/index.tsx

```diff
@@ -80,6 +88,9 @@ export default function CioQuiz(props: IQuizProps) {
         <QuizContext.Provider value={contextValue}>
           {state.quiz.results && <ResultContainer options={resultsPageOptions} />}
           {state.quiz.currentQuestion && <QuizQuestions />}
+          {state.quiz.currentQuestion && (
+            <ControlBar ctaButtonText={questionData?.cta_text || undefined} />
```

Comment:
Typescript wasn't happy with `questionData?.cta_text` here, let me know if there is a better way to do this